math/lcm_sum.cpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * @file
+ * @brief An algorithm to calculate the sum of LCM: \f$\mathrm{LCM}(1,n) + \mathrm{LCM}(2,n) + .. + \mathrm{LCM}(n,n)\f$
+ * @details An algorithm to calculate the sum of LCM: \f$\mathrm{LCM}(1,n) + \mathrm{LCM}(2,n) + .. + \mathrm{LCM}(n,n)\f$ where \f$\mathrm{LCM}(i,n)\f$
+ * denotes the Least Common Multiple of the integers i and n. For n greater than or equal to 1.
+ * The value of the sum is calculated by formula:
+ * \f[
+ * \sum\mathrm{LCM}(i, n) = \frac{1}{2} \left[\left(\sum (d * \mathrm{ETF}(d)) + 1\right) * n\right]
```

Comment:
what is ETF in the equation?

Reply:
ETF(i) represents the Euler totient function of i.